### ISSUE TITLE
Empty query string creates a map with a enpty key and value on Input

### ISSUE THREAD

**Issue by adaojunior** [dart-bridge/framework#40]
A empty query string will create a empty key and value on Input.
example:

```
/homepage
...

class PagesController {
  index(Input input) {
    print(input);
    return template('index').withScript('main');
  }
}

```

It will print a `Input({: })`.

**Comment by adaojunior** [comment]
Seems like it is a issue with **formler**
I created a issue https://github.com/Daegalus/formler/issues/4

**Comment by emilniklas** [comment]
Looking at that issue it seems Formler has been fixed. I'll verify that today

**Comment by emilniklas** [comment]
Because of dependency inconsistencies we're stuck on formler `0.0.8`, however I think it's because formler imports the unittest package as a normal dependency. I've created a PR to try to solve this issue;

Daegalus/formler#5

**Comment by emilniklas** [comment]
This is solved in the upcoming Formler 0.1.2. Currently using git dependency in pubspec.
